src/components/Hashcat/Hashcat.tsx

```diff
@@ -41,8 +18,22 @@ interface FormValuesType {
     maxPwdLen: number;
 }
 
-const inputType = ["Hash Value", "File"];
-const attackMode = ["Straight", "Brute-force", "Hybride Wordlist + Mask"];
+const title = "Hashcat";
+const description_userguide = `
+    Hashcat is an advanced password recovery tool that provides brute-force attacks that are conducted with the hash values of passwords that are either guessed or applied by the tool. DDT currently supports 3 attack modes including Straight, Brute-force, and Hybrid Wordlist + Mask.
+
+    Step-by-Step Guide:
+    1. Pick an attack mode.
+    2. Input hash type and hash algorithm code.
+    3. Input the hash value.
+    4. Input the password file.
+    5. Add additional commands as needed.
+    6. Click Scan to commence.
+    7. View the output below.
```

Comment:
You use Step 1:... , Step 2:,...
```suggestion
    Step-by-Step Guide:
    Step 1: Pick an attack mode.
    Step 2: Input hash type and hash algorithm code.
    Step 3: Input the hash value.
    Step 4: Input the password file.
    Step 5: Add additional commands as needed.
    Step 6: Click Scan to commence.
    Step 7: View the output below.
```